.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+    - "2.7"
+before_install:
+    - "export DISPLAY=:99"
+    - "sh -e /etc/init.d/xvfb start"
+install:
+    - "sh install_test_deps.sh"
+    - "python setup.py develop"
```

Comment:
It took me a while to realize what you mean, but I think I get it now. This just ensures that the build fails if you forget to include some directories in the `package_data` in `setup.py`, but assuming your `package_data` is correct, there should be no difference, right?